pytorch_lightning/accelerators/gpu.py

```diff
@@ -76,6 +81,9 @@ def auto_device_count() -> int:
         """Get the devices when set to auto."""
         return torch.cuda.device_count()
 
+    def teardown(self) -> None:
+        torch.backends.cudnn.benchmark = False
```

Comment:
It sounds overkill but maybe this should restore the previous value before the Trainer